duckduckgo/macos-browser
# Special bundle id and app icon for the debug configuration

Task/Issue URL: https://app.asana.com/0/1199230911884351/1199945828384232/f

**Description**:
PR adds a special bundle id and app icon for DEBUG configuration

**Steps to test this PR**:
Set DEBUG/RELEASE configuration for the current scheme and:

- make sure the app icon is appropriate. 
- app bundles are different (easy to test with session restoring)


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
